src/OperationProcessor.ts

```diff
@@ -113,33 +151,56 @@ function earlier (ts1: OperationTimestamp, ts2: OperationTimestamp): boolean {
 }
 
 /**
- * Information about a write operation relevant for the DID cache, a subset of the properties exposed by
- * WriteOperation.
+ * Information about a write operation relevant for maintaining OperationProcessor state.
  */
 interface OperationInfo {
   readonly batchFileHash: string;
   readonly type: OperationType;
   readonly timestamp: OperationTimestamp;
+  readonly parent?: OperationHash;
```

Comment:
Nit: parent type is more appropriate/consistent to be `VersionId` (like `missingAncestor)?

Reply:
Sure, fixed.